recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "papermill" %}
-{% set version = "2.4.0" %}
+{% set version = "2.5.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Given that the published sdist is busted we can't use it here, at least not for this release. For this release we can instead use the GitHub archive:
```suggestion
  # FIXME: Revert GitHub archive change when sdist is correctly packaged
  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  url: https://github.com/nteract/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
```